algorithms/ar-bsrt/python2/bubble_sort.py

```diff
@@ -0,0 +1,21 @@
+# Bubble sort function
+def bubbleSort(arr):
+	n = len(arr)
+	for i in range(n-1):
+		for j in range(0, n-i-1):
+			# Swap if current element is greater than next element
+			if (arr[j]>arr[j+1]):
+				arr[j], arr[j+1] = arr[j+1], arr[j]
+
+# Function to print an array
+def printArray(arr):
+	for i in range(len(arr)):
```

Comment:
Codacy found an issue: [Consider using enumerate instead of iterating with range and len](https://app.codacy.com/gh/FOSS-UCSC/FOSSALGO/pullRequest?prid=6280328)